src/lib.rs

```diff
@@ -68,6 +69,8 @@
 //! sounds to play simultaneously, you should create multiple [`Sink`]s.
 //!
 //! The [`Sink`] type also provides utilities such as playing/pausing or controlling the volume.
+//! 
+//! **Please note that the [`Sink`] requires the [`OutputStream`], thus ensuring that it has to live as long as the sink.**
```

Comment:
One small suggestion, maybe: "make sure that the outputstream is not dropped before the sink" is better then "thus ensuring that it has to live as long as the sink"?